### PR TITLE
Fix Title and PR links in site/_posts/2017-04-17-Fine-tuning-Fine.md

### PR DESCRIPTION
**Pull request by AparnaKarve**
@JPrause @cybette I happened to notice in http://manageiq.org/blog/2017/04/Fine-tuning-Fine/ that the PR links were not showing up correctly.

Hence I have created this PR to fix the links.

I was using the Github `Preview` feature before which shows the links just fine, probably why I didn't realize that there was an issue.
While at it, I've also shortened the title to simply `And Finally . . . Fine-tuning Fine`

Sorry about this PR.
Rookie blogger mistake I suppose :)

### CODE REVIEW

File: site/_posts/2017-04-17-Fine-tuning-Fine.md [hunk 1]
```diff
@@ -32,68 +32,62 @@ I'll do my best to bring you some of the **`Fine`** highlights of the repositori
 
 ## Some Fine 'Finishing' Touches
 
-- [Laura Galis](https://github.com/lgalis) fixed the PDF download issue for Ansible Tower Job Templates in https://github.com/ManageIQ/manageiq-ui-classic/pull/523
+- [Laura Galis](https://github.com/lgalis) fixed the PDF download issue for Ansible Tower Job Templates in [ManageIQ/manageiq-ui-classic#53](https://github.com/ManageIQ/manageiq-ui-classic/pull/523)
```
Comment (cybette) [inline site/_posts/2017-04-17-Fine-tuning-Fine.md]: 53 -> 523 (2 is missing)

File: site/_posts/2017-04-17-Fine-tuning-Fine.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Last Week in ManageIQ: And Finally . . . Fine-tuning Fine"
```
Comment (cybette) [inline site/_posts/2017-04-17-Fine-tuning-Fine.md]: I'd prefer if the "Last Week in ManageIQ is left in, to be consistent with [most of the posts](http://manageiq.org/blog/tags/LWIMIQ/) so far. Only bad example is last week's post :sweat_smile:

**Comment by cybette** [comment]
> Rookie blogger mistake I suppose :)

Minor stuff, I didn't think about the links either when I was previewing them! Thanks for the changes :)

File: site/_posts/2017-04-17-Fine-tuning-Fine.md (hunk 1 above)
Comment (AparnaKarve) [inline site/_posts/2017-04-17-Fine-tuning-Fine.md]: Ok, fixed that. Thanks :)

File: site/_posts/2017-04-17-Fine-tuning-Fine.md (hunk 2 above)
Comment (AparnaKarve) [inline site/_posts/2017-04-17-Fine-tuning-Fine.md]: And restored the title to `Last Week in ManageIQ: ....`

**Pull request merged**